ledger/acctupdates.go

```diff
@@ -134,13 +134,16 @@ type modifiedResource struct {
 	ndeltas int
 }
 
-// A modifiedValue represents the value that has been modified since the
-// persistent state stored in the account DB (i.e., in the range of rounds
-// covered by the accountUpdates tracker).
+// A modifiedValue represents a kv store change since the persistent state
+// stored in the DB (i.e., in the range of rounds covered by the accountUpdates
+// tracker).
 type modifiedValue struct {
```

Comment:
this applies to the regular box branch, but maybe a more KV specific name than `modifiedValue`?

Reply:
I hate this name as well.  I didn't want to call it `modifiedKeyValue` because I thought that would imply it contained a key. But I suppose you could think of the `Key` in `modifiedKeyValue` as simply an adjective.

An alternate is something like `modifiedKValue`?

Reply:
Ok, I'm going with that, I'll do it in a new PR since this is merged to the branch already.